### PR TITLE
Added AlertResendDelay to Prometheus resource

### PR DESCRIPTION
**Pull request by zuzzas**
Added the option to Prometheus custom object, since there wasn't one.

Fixes https://github.com/coreos/prometheus-operator/issues/2253

### CODE REVIEW

File: pkg/apis/monitoring/v1/types.go [hunk 1]
```diff
@@ -118,6 +118,8 @@ type PrometheusSpec struct {
 	ScrapeInterval string `json:"scrapeInterval,omitempty"`
 	// Interval between consecutive evaluations.
 	EvaluationInterval string `json:"evaluationInterval,omitempty"`
+	// Minimum amount of time to wait before resending an alert to Alertmanager.
+	AlertResendDelay string `json:"alertResendDelay,omitempty"`
```
Comment (brancz) [inline pkg/apis/monitoring/v1/types.go]: as there are multiple `-rules.alert` flags, could you add two config structs for this?

basically so we end up with

```
rules:
  alert:
    resendDelay: ...
```

**Comment by brancz** [comment]
just one comment otherwise this is looking pretty good, thanks for taking care of it!

**Comment by zuzzas** [comment]
@brancz, it seems that everything is per your desires. Can you review the naming of structs and comments, please? Never been good with those thing.

**Comment by brancz** [comment]
Looks good! Thanks! :+1:

**Pull request merged**